### PR TITLE
Support multiple compose files

### PR DESCRIPTION
**Pull request by ilkkao**
YAML arrays are not supported but multiline is

Idea is re-used from https://github.com/isbang/compose-action

### CODE REVIEW

File: src/context.ts [hunk 1]
```diff
@@ -124,7 +124,7 @@ export async function getContext(): Promise<Context> {
   };
 
   core.saveState('isPost', post);
-  core.saveState('composeFile', context.composeFile);
+  core.saveState('composeFiles', context.composeFiles);
```
Comment (sjagoe) [inline src/context.ts:127]: Should we put an explicit `JSON.stringify` in here to match the `JSON.parse` below?

Comment (ilkkao) [reply src/context.ts:127]: This is the part I didn't fully understand. `composeArguments` below doesn't have JSON.stringify but has JSON.parse on reading step. I copied that. But if these state variables are store as strings, yes then stringify() is needed.

Comment (ilkkao) [reply src/context.ts:127]: @sjagoe found it from the core toolkit docs

```
 * @param     value    value to store. Non-string values will be converted to a string via JSON.stringify
```

So `JSON.stringify()` is not needed. This should be correct.

Comment (ilkkao) [reply src/context.ts:127]: `getState` doesn't have automatic conversion so there `JSON.parse()` is needed.

**Review by sjagoe — APPROVED** [review]

**Pull request merged**